File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution(object):

    def isPalindrome(self, s):

        s = s.lower()

        new_s = ""

        for c in s:
            if c.isalnum():
                new_s += c.lower()

        s = new_s

        r = s[::-1]

        if r == s:
            return True
        else:
            return False

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string new_s = "";

        for(char c : s) {
            if(isalnum(c)) {
                new_s += tolower(c);
            }
        }

        string r = new_s;
        reverse(r.begin(), r.end());

        if(r == new_s)
            return true;
        else
            return false;
    }
};